distribution/macos/create_app_bundle.sh

```diff
@@ -6,31 +6,31 @@ PUBLISH_DIRECTORY=$1
 OUTPUT_DIRECTORY=$2
 ENTITLEMENTS_FILE_PATH=$3
 
-APP_BUNDLE_DIRECTORY=$OUTPUT_DIRECTORY/Ryujinx.app
+APP_BUNDLE_DIRECTORY="$OUTPUT_DIRECTORY"/Ryujinx.app
```

Comment:
I'd like to put the complete path in quotes.
```suggestion
APP_BUNDLE_DIRECTORY="$OUTPUT_DIRECTORY/Ryujinx.app"
```